python/sglang/bench_serving.py

```diff
@@ -587,6 +587,8 @@ async def benchmark(
     else:
         print("Initial test run completed. Starting main benchmark run...")
 
+    time.sleep(1.5)
```

Comment:
Just curious, why do we need to sleep for 1.5 seconds here?

Reply:
@zhyncs to separate the two phases (warmup and benchmark) better. Useful for logging and other things.